core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -124,6 +124,51 @@ <h3>Update username</h3>
     </form>
   </div>
 </md-card>
+
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h2>Wipeout</h2>
+  <h3>Number of pending deletion requests</h3>
+  <div class="oppia-query-form">
+    <form class="form"
+          ng-submit="$ctrl.getNumberOfPendingDeletionRequestModels()">
+      <div class="form-group row">
+        <input type="submit" value="Get number of PendingDeletionRequestModels">
+      </div>
+    </form>
+  </div>
+  <h3>Delete account</h3>
+  <div class="oppia-query-form">
+    <form class="form"
+          ng-submit="$ctrl.deleteAccount()">
+      <div class="form-group row">
+         (the user needs to request the deletion before this can be run)
+      </div>
+      <div class="form-group row">
+        <label>Email:</label>
+        <input type="text" ng-model="$ctrl.email" required="true">
+      </div>
+      <div class="form-group row">
+        <input type="submit" value="Delete account">
+      </div>
+    </form>
+  </div>
+  <h3>Verify account deleted</h3>
+  <div class="oppia-query-form">
+    <form class="form" ng-submit="$ctrl.verifyAccountDeleted()">
+      <div class="form-group row">
+         (the user needs to be deleted)
```

Comment:
What happens if this doesn't hold?

Reply:
If the user didn't request the deletion: an exception that there is no pending deletion request for this user.

If the user requested the deletion but it wasn't yet run: it will return a message that the deletion was not yet run.

Reply:
Done, removed.